api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,25 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
-  createInstitutions(system)
-  createLocalHmdaEventProcessor(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started validator at ${actor.path}")
+    }
+
+  (supervisor ? FindActorByName(LocalHmdaEventProcessor.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started event processor at ${actor.path}")
+    }
+
+  (supervisor ? FindActorByName(InstitutionPersistence.name))
```

Comment:
Now that we are using `FindActorByName` for this actor whenever it is being used in the api do we still want to start a copy of it here?